CMake/platforms/amiga.cmake

```diff
@@ -20,7 +20,7 @@ add_definitions(-Dfseeko=fseek)
 
 list(APPEND DEVILUTIONX_PLATFORM_LINK_LIBRARIES ZLIB::ZLIB)
 if(NOT WARPOS)
-  list(APPEND DEVILUTIONX_PLATFORM_LINK_LIBRARIES -ldebug)
+  list(APPEND DEVILUTIONX_PLATFORM_LINK_LIBRARIES -ldebug -fbbb=-)

```

Comment:
Have you tested which optimization is breaking things?

https://eab.abime.net/showthread.php?t=85474&page=51

How does the issue materialize?

Reply:
Crashes system without it. I will report this to Amiga gcc maintainer. I did that many times before and he fixed it.

Reply:
Although this is wrong place for this switch.